types/p5/tslint.json

```diff
@@ -0,0 +1,24 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
In new package definitions we require this to be only extends clause and no edits here are allowed after generating it through dts-gen

Reply:
I managed to remove some custom rules but the two that remains are extremely difficult to fix.
The definitions are generated by a custom tool that reads the project's YUIDoc data (basically JSDoc).
That data does not contain the necessary information (yet?) to emit a more specific type than `Function` (that's why we need a custom "ban-types") and I can't come up with a good way for our tool to merge signatures since there can be different comments for each overload (that's why we need to disable "unified-signatures").
Fixing these errors by hand is not really an option since it would be quite labour intensive and risks being undone when we want to run the tool again as changes to the documentation/methods are made.

As stated in the first comment I also need some guidance on how to include some files in the generated project without having them automatically imported (that would pollute global scope + augment the p5 instance with things that are not available unless those files are imported).

Reply:
I managed to remove the "ban-types" override as well 🎉!
I really hope we can leave "unified-signatures" in as I don't have an algorithm for merging the signatures automatically, especially not if we want to preserve the comments.
Or should `// tslint:disable-next-line:unified-signatures` be generated where it's needed?

Reply:
The individual suppressions would probably be preferable; but since this is generated and nobody's meant to edit it by hand, it probably doesn't matter.